spec/lib/internal_metric/global_spec.rb

```diff
@@ -113,13 +113,15 @@ module DiscoursePrometheus::InternalMetric
       end
     end
 
-    it "can collect pg_seq metric" do
-      3.times { Fabricate(:user) }
-
+    it "can collect pg_seq metric and caches the sequence" do
       metric.collect
 
       expect(metric.postgres_highest_sequence).to be_a_kind_of(Hash)
-      expect(metric.postgres_highest_sequence[{ db: "default" }]).to be > 3
+      expect(metric.postgres_highest_sequence[{ db: "default" }]).to be_present
```

Comment:
Just checking the presence here because there is actually a subtle bug here where the highest sequence may not be coming from the `users` table depending on the state of each database.